drivers/spi/spi_xec_qmspi_ldma.c

```diff
@@ -1242,14 +1095,14 @@ static int qmspi_xec_init(const struct device *dev)
 	qdata->xfr_len = 0;
 #endif
 
-	int ret = xec_qspi_config_port();
+	z_mchp_xec_pcr_periph_sleep(cfg->pcr_idx, cfg->pcr_pos, 0);
```

Comment:
Does 17xx soc header have MCHP_PCR_SLEEP_DIS macros?

Reply:
172x soc headers do not have these macros. The code reviewers (at the time the 172x soc was checked in) demanded no macros for register access. They wanted the functionality in drivers.